GDJS/Runtime/CustomRuntimeObject.ts

```diff
@@ -172,6 +172,13 @@ namespace gdjs {
           newObjectData.animatable || []
         );
       }
+      if (oldObjectData.variant !== newObjectData.variant) {
+        this._reinitializeRenderer();
+        this._initializeFromObjectData(newObjectData);
+
+        // The generated code calls the onCreated super implementation at the end.
+        this.onCreated();
```

Comment:
When changing of variant the instance is like a new instance. We let the events initialize it.

Reply:
Reword the comment to be more like this then:

```
When changing the variant, the instance is like a new instance. We call again `onCreated` at the end, like done by the constructor the first time it's created.
```